networkx/algorithms/community/quality.py

```diff
@@ -222,7 +223,10 @@ def modularity(G, communities, weight="weight", resolution=1):
     .. [4] M. E. J. Newman, "Equivalence between modularity optimization and
        maximum likelihood methods for community detection"
        Phys. Rev. E 94, 052315, 2016. https://doi.org/10.1103/PhysRevE.94.052315
-
+    .. [5] Blondel, V.D. et al. "Fast unfolding of communities in large
+       networks" J. Stat. Mech 10008, 1-12 (2008).
+       https://doi.org/10.1088/1742-5468/2008/10/P10008
+       
```

Comment:
This is necessary to pass the lint job - normally I'd just push this up but that's not possible since the PR originates from `main`; sorry for the nittyness!

```suggestion
```